CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
   - Added support to Interconnect actions
   - Added support to Logical switch group actions
   - Added support to Firmware bundle actions
+  - Added support to Storage system `:edit_credentials` and `:add_if_missing`
+  - Changed implementation of Storage system `:add` and `:remove`
```

Comment:
These changes are relevant to the user? If so, please specify what are those, otherwise, remove this entry.